tools/block-generator/generator/generate.go

```diff
@@ -887,285 +785,21 @@ func (g *generator) generateAssetTxnInternalHint(txType TxTypeID, round uint64,
 	return
 }
 
-// ---- 3. App Transactions ----
-
-func (g *generator) generateAppTxn(round uint64, intra uint64) ([]txn.SignedTxn, []txn.ApplyData, uint64 /* nextIntra */, uint64 /* appID */, error) {
```

Comment:
This was moved to `generate_apps.go`. Additionally:
* it no longer returns `[]txn.ApplyData` or deals with an `txn.ApplyData`
* `intra` is updated using a new function `countEffects()`